bin/pre-commit-hook.js

```diff
@@ -79,3 +80,32 @@ if ( phpLintResult && phpLintResult.status ) {
 	);
 	process.exit( 1 );
 }
+
+let phpcbfResult, phpcsResult;
+if ( phpcsFiles.length > 0 ) {
+	phpcbfResult = spawnSync( 'vendor/bin/phpcbf', [...phpcsFiles], {
```

Comment:
Do we want to `git add` the files after making the changes, so the changes get included in the commit?

Reply:
We don't need to. The check runs against `git diff --cached` which only returns staged files, then the files are modified by the precommit hook, then they're commited. Since we're only checking staged file pre-commit, then committing (assuming the phpcs check doesn't fail), no need to `git add`.